File: Java/Moneda.java
class Moneda{
	private int idMoneda;
	private String codigoISO;
	private String simbolo;
	
	public Moneda(int idMoneda, String codigoISO, String simbolo) {
    this.idMoneda = idMoneda;
    this.codigoISO = codigoISO;
    this.simbolo = simbolo;
	}

	public int getIdMoneda() {
		return idMoneda;
	}

	public void setIdMoneda(int idMoneda) {
		this.idMoneda = idMoneda;
	}

	public String getCodigoISO() {
		return codigoISO;
	}

	public void setCodigoISO(String codigoISO) {
		this.codigoISO = codigoISO;
	}

	public String getSimbolo() {
		return simbolo;
	}

	public void setSimbolo(String simbolo) {
		this.simbolo = simbolo;
	}
}

File: web/EconomixSolution/EconomixModel/Model/tesoreria/Moneda.cs
using System.Text.Json.Serialization;

namespace EconomixModel.Model;

public class Moneda
{
    [JsonPropertyName("idMoneda")]
    public int IdMoneda { get; set; }

    [JsonPropertyName("codigoISO")]
    public string CodigoISO { get; set; } = string.Empty;

    [JsonPropertyName("simbolo")]
    public string Simbolo { get; set; } = string.Empty;

    [JsonPropertyName("nombre")]
    public string Nombre { get; set; } = string.Empty;

    [JsonPropertyName("descripcion")]
    public string Descripcion { get; set; } = string.Empty;

    [JsonPropertyName("activa")]
    public bool Activa { get; set; } = true;
}
